.changes/v3.6.0/793-improvements.md

```diff
@@ -0,0 +1,2 @@
+* `resource/vcd_nsxt_edgegateway` and `resource/vcd_nsxt_edgegateway` support VDC groups via new
+  field `owner_id` replacing `vdc` [GH-793]
```

Comment:
Tiny nit, but these two lines should probably be in a single line (as is usual with old entries).

Actually, I have noticed this new-line breaking in new changelog entries in our PRs just recently. Maybe formatter changed its behavior?